pkglistgen/tool.py

```diff
@@ -564,7 +613,8 @@ def solve_project(
                         module.solved_packages[arch].pop(p, None)
 
         self._collect_unsorted_packages(modules, self.groups.get('unsorted'))
-        return self.write_all_groups()
```

Comment:
why did "write_all_groups" get deleted here?

Reply:
I see, it got moved elsewhere.

Reply:
because it only happens now when the old product-builder products get generated.